subsys/bluetooth/mesh/blob_cli.c

```diff
@@ -1550,9 +1550,9 @@ int bt_mesh_blob_cli_send(struct bt_mesh_blob_cli *cli,
 		return -ENODEV;
 	}
 
-	LOG_DBG("\n\tblock size: %u\n\tchunk size: %u\n"
+	LOG_DBG("\n\tblock size log: %u\n\tchunk size: %u\n"
 		"\tblob size: %u\n\tmode: %x",
-		(1 << cli->xfer->block_size_log), cli->xfer->chunk_size,
```

Comment:
```
this will add little bit more code when debug option is enabled
```
I really do not like the approach when we try to think for the compiler. Lets it does its work. 
I'd prefer to see in debug logs understandable values rather to go in specification and to look for formula to understand logarithm from what.

Reply:
This definitely requires uint64_t to do bit shift safely. uint64_t will require %llu specified, which in turns if I understand correctly will require `CONFIG_MINIMAL_LIBC_LL_PRINTF` option enabled to print the value correctly. So I'd prefer to print logarithmic value here.

Reply:
> Up to you, of course. However, not a lot of sense in debug output if it doesn't help you understand instant behavior. I do not see big problem to link debug build with `CONFIG_MINIMAL_LIBC_LL_PRINTF`. Even more, `uint64_t` has been used already in many places and practically it will cost nothing. For example: https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/mesh/blob_srv.c#L436-L437

This is a different thing IMO. No problem in uint64_t in general, but in printing the value.